articles/architecture-scenarios/implementation/b2c/authentication.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Authentication
+description: Understand how authentication works in your CIAM implementation.
+toc: true
+topics:
+    - b2c
+    - ciam
+    - authentication
+    - universal-login
+contentType: concept
+useCase:
+    - authentication
+---
+# Authentication
+
+In order to provide services to your users, you must be able to identify who those users are. This process is called authentication. You must provide your users with a way to verify that they are who they are, and there are a number of ways to do this: via social media accounts, username/password, and passwordless authentication to name but a few. In many cases it is also recommended that you go beyond a first factor for authenticating the user add a second factor as well.
+
```

Comment:
There was a "best practice" callout here that now seems to be missing (https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.pi8vu9ftyaza).

Reply:
Added it back in.